spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -948,6 +948,10 @@ def getRegistrationSaltSh(productName):
     MINION_SERVICE="venv-salt-minion"
 fi
 
+if [ -n "$SNAPSHOT_ID" ]; then
+    test -d "$MINION_CONFIG_DIR" || mkdir -p "$MINION_CONFIG_DIR"
```

Comment:
I'm not sure I understand here, but if the only problem is that we could re-execute the bootstrap script multiple times, shouldn't we just continue in the previous transaction?

In `new_transaction`, we use `/usr/sbin/tukit -q open`, we could use `/usr/sbin/tukit --continue --discard -q open` instead?

Reply:
There might have been transactions in between the one that added `etc/venv-salt-minion/` and the latest. I didn't want to search all the transactions, thought it was cleaner to "onboard in a new transaction", no matter what happened before.

Reply:
Hmm, that sounds about right. I suppose this would be situational. Perhaps at some point we could let the users decide whether they want a new transaction or continue in previous one.